cmd/cloud/dbt_test.go

```diff
@@ -0,0 +1,154 @@
+package cloud
+
+import (
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+
+	astrocore "github.com/astronomer/astro-cli/astro-client-core"
+	astroplatformcore "github.com/astronomer/astro-cli/astro-client-platform-core"
+	astroplatformcore_mocks "github.com/astronomer/astro-cli/astro-client-platform-core/mocks"
+	cloud "github.com/astronomer/astro-cli/cloud/deploy"
+	testUtil "github.com/astronomer/astro-cli/pkg/testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestDbtDeploy_PickDeployment(t *testing.T) {
```

Comment:
same as before, might want to restructure these tests to use the new `testify/suite`